### PR TITLE
feat(Title): add tokens for Title2 and create Title3

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1484)

### CODE REVIEW

File: src/header.tsx [hunk 1]
```diff
@@ -94,9 +96,17 @@ export const Header: React.FC<HeaderProps> = ({
                     <Stack space={8}>
                         {pretitle && renderRichText(pretitle, {color: vars.colors.textPrimary})}
                         {small ? (
-                            <Text4 regular role="heading" aria-level={2}>
+                            <Text
+                                mobileSize={textPresets.title2.size.mobile}
```
Comment (atabel) [inline src/header.tsx:100]: why are you using title2 preset in header?

Comment (marcoskolodny) [reply src/header.tsx:100]: It is requested in Jira's issue :)

File: src/title.tsx [hunk 2]
```diff
@@ -60,12 +60,35 @@ export const Title1 = ({children, as = 'h3', id, right, dataAttributes}: TitlePr
...
+            title={
+                <Text
+                    as={as}
+                    id={id}
+                    mobileSize={textPresets.title2.size.mobile}
+                    desktopSize={textPresets.title2.size.desktop}
+                    weight={textPresets.title2.weight}
+                    mobileLineHeight={textPresets.title2.lineHeight.mobile}
+                    desktopLineHeight={textPresets.title2.lineHeight.desktop}
+                    dataAttributes={{'component-name': 'Title2', ...dataAttributes}}
```
Comment (pladaria) [inline src/title.tsx:75]: IMO, this should be moved to the wrapper. The component is not just this text

**Review by pladaria — APPROVED** [review]

File: src/title.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/title.tsx]: good call. Fixed

File: src/header.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/header.tsx]: Discussed offline. We decided to use `Title2` and `Title3` for small and large header, respectively.

**Review by aweell — APPROVED** [review]

File: src/header.tsx [hunk 3]
```diff
@@ -96,21 +95,9 @@ export const Header: React.FC<HeaderProps> = ({
...
-                                aria-level={2}
-                            >
-                                {title}
-                            </Text>
+                            <Title2 aria-level={2}>{title}</Title2>
                         ) : (
-                            <Text6 role="heading" aria-level={2}>
-                                {title}
-                            </Text6>
+                            <Title3 aria-level={2}>{title}</Title3>
```
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/header.tsx]: fixed

File: src/header.tsx [hunk 4]
```diff
@@ -93,15 +94,7 @@ export const Header: React.FC<HeaderProps> = ({
...
-                        {small ? (
-                            <Text4 regular role="heading" aria-level={2}>
-                                {title}
-                            </Text4>
-                        ) : (
-                            <Text6 role="heading" aria-level={2}>
-                                {title}
-                            </Text6>
-                        )}
+                        {small ? <Title2>{title}</Title2> : <Title3>{title}</Title3>}
```
Comment (atabel) [inline src/header.tsx:97]: To keep the heading level we should use `as="h2"` here
```suggestion
                        {small ? <Title2 as="h2">{title}</Title2> : <Title3 as="h2">{title}</Title3>}
```

Comment (marcoskolodny) [reply src/header.tsx]: fixed

**Review by atabel — APPROVED** [review]

**Pull request merged**